test-suite/djinni/date.yaml

```diff
@@ -23,6 +23,8 @@ java:
   reference: true
   generic: true
   hash: '%s.hashCode()'
+  writeToParcel: 'out.writeString(%s.toString())'
+  readFromParcel: 'new %s(in.readString())'
```

Comment:
This is using a deprecated constructor. Wouldn't it be more efficient to use `getTime()` and `Date(long)` instead?